packages/angular/build/src/builders/application/schema.json

```diff
@@ -125,6 +125,34 @@
             "type": "string"
           },
           "default": []
+        },
+        "sassOptions": {
```

Comment:
nit: Thoughts on shortening to `sass` since it's already inside something ending with `Options`?